packages/utils/regl-renderer/types/controls/orbitControls.d.ts

```diff
@@ -0,0 +1,125 @@
+import { Geometry } from '@jscad/modeling/src/geometries/types'
```

Comment:
Is this import necessary? I would be surprised if so.

Reply:
Not necessary, but it's nice... for example when you use entitiesFromSolids() it will let you know that the return  of that function is type Geometry[], which will show up in an IDE with correct typed properties like polygons and transforms. If you type it as any then you don't get autocomplete on properties or type checking for other functions like zoomToFit.

Reply:
Ok